src/image.rs

```diff
@@ -619,6 +619,7 @@ pub trait GenericImageView {
     /// Returns the pixel located at (x, y)
     ///
     /// This function can be implemented in a way that ignores bounds checking.
+    #[allow(clippy::missing_safety_doc)]
```

Comment:
These ones just make us forget that a safety doc should actually be added.

```suggestion
    /// # Safety
    ///
    /// The coordinates must be [`in_bounds`] of the image.
    ///
    /// [`in_bounds`]: #method.in_bounds
```

Reply:
Sure, I've accepted your commit suggestion.